crates/hc_sandbox/Cargo.toml

```diff
@@ -34,6 +34,7 @@ holo_hash = { version = "^0.6.0-dev.0", path = "../holo_hash", features = [
   "kitsune2",
 ] }
 holochain_chc = { version = "^0.3.0-dev.0", path = "../holochain_chc", optional = true }
+holochain_client = { version = "^0.7.0-rc.0", path = "../client" }
```

Comment:
Do I need to consider something here when specifying a version that it gets updated properly with each release?